pkg/api/v1/defaults.go

```diff
@@ -111,6 +111,12 @@ func SetDefaults_Firmware(obj *Firmware) {
 	}
 }
 
+func SetDefaults_Rng(obj *Rng) {
+	if obj.Source == "" && !obj.Disabled {
```

Comment:
I think this should just always set the obj.Source = "/dev/urandom" - no need to check if the obj.Source
And SetDefaults_Rng should be called only if vmi.Spec.Domain.Devices.Rng != nil